### ISSUE TITLE
Cannot parse CREATE INDEX on a json text value

### ISSUE THREAD

**Issue by oomathias** [sbdchd/squawk#59]
```sql
CREATE INDEX CONCURRENTLY IF NOT EXISTS "idx_a_foo_bar" ON "a" ((foo->>'bar'));
```

```sh
Err(
    CheckSQL(
        ParsingSQL(
            JsonParse(
                "invalid type: map, expected a string at line 1 column 225",
            ),
        ),
    ),
)
```

The double parentheses are required in `((foo->>'bar'))` but it looks like it's parsed as a `map` instead of a `string`.

**Comment by sbdchd** [comment]
Thanks for the report! looking into it

**Comment by sbdchd** [comment]
Unclosing this since I haven't deployed this version yet

**Comment by sbdchd** [comment]
Should be fixed on the latest version `0.5.0` which is available on NPM or via the github releases
